src/relentless/optimize/objective.py

```diff
@@ -288,12 +292,13 @@ class RelativeEntropy(ObjectiveFunction):
 
     """
 
-    def __init__(self, target, simulation, potentials, thermo, extensive=False):
+    def __init__(self, target, simulation, potentials, thermo, extensive=False, T=None):
```

Comment:
The docstring for this class needs to be updated to reflect the new arguments that can be accepted, and to explain the differences in calculations (and what is supported or not).